manager/manager/utils.py

```diff
@@ -505,6 +505,57 @@ def jira_ticket(request_data):
     )
 
 
+def update_time_loss(jira_id: int, add_time_loss: float = 0.0) -> Response:
+    """Connect to the Rubin Observatory JIRA Cloud REST API to
+    update a jira ticket's specific field
+
+    Params
+    ------
+    jira_id: int
+        Jira ID
+
+    Returns
+    -------
+    Response
+        define response here
+    """
+    headers = {
+        "Authorization": f"Basic {os.environ.get('JIRA_API_TOKEN')}",
+        "content-type": "application/json",
+    }
+    get_url = (
+        f"https://{os.environ.get('JIRA_API_HOSTNAME')}/rest/api/latest/issue/{jira_id}"
+    )
+    response = requests.get(get_url, headers=headers)
+    existent_time_loss = (
+        response.json().get("customfield_10106", 0.0)
+        if response.status_code == 200
+        else 0.0
+    )
+    jira_payload = {
+        "fields": {
+            "customfield_10106": float(existent_time_loss + add_time_loss),
+        },
+    }
+    put_url = f"https://{os.environ.get('JIRA_API_HOSTNAME')}/rest/api/latest/issue/{jira_id}/"
+    response = requests.put(put_url, json=jira_payload, headers=headers)
+
+    if response.status_code == 200 or response.status_code == 204:
+        return Response(
+            {
+                "ack": "Jira field updated",
+                "url": f"https://{os.environ.get('JIRA_API_HOSTNAME')}/browse/{jira_id}",
+            },
+            status=200,
+        )
+    return Response(
+        {
+            "ack": "Jira field could not be updated",
```

Comment:
We should specify here the time_lost field could not be updated.
```suggestion
            "ack": "Jira time_lost field could not be updated",
```